src/server/scripts/Kalimdor/CavernsOfTime/CullingOfStratholme/npc_arthas.cpp

```diff
@@ -0,0 +1,604 @@
+                    break;
+            }
+        }
+
+        void MovementInform(uint32 type, uint32 id)
+        {
+            if (type != POINT_MOTION_TYPE)
+                return;
+            switch (id)
+            {
+                case RP1_UTHER_WALK_TO:
+                    if (Creature* uther = me->FindNearestCreature(NPC_UTHER, 100.0f, true))
+                    {
+                        uther->SetFacingToObject(me);
+                        me->SetFacingToObject(uther);
+
+                        events.ScheduleEvent(RP1_EVENT_ARTHAS1, Seconds(1));
+                        events.ScheduleEvent(RP1_EVENT_UTHER1, Seconds(5));
+                        events.ScheduleEvent(RP1_EVENT_ARTHAS2, Seconds(12));
+                        events.ScheduleEvent(RP1_EVENT_ARTHAS3, Seconds(22));
+                        events.ScheduleEvent(RP1_EVENT_UTHER2, Seconds(33));
+                        events.ScheduleEvent(RP1_EVENT_ARTHAS4, Seconds(35));
+                        events.ScheduleEvent(RP1_EVENT_UTHER3, Seconds(39));
+                        events.ScheduleEvent(RP1_EVENT_ARTHAS5, Seconds(45));
+                        events.ScheduleEvent(RP1_EVENT_UTHER4, Seconds(51));
+                        events.ScheduleEvent(RP1_EVENT_ARTHAS6, Seconds(57));
+                        events.ScheduleEvent(RP1_EVENT_UTHER5, Seconds(60));
+                        events.ScheduleEvent(RP1_EVENT_ARTHAS7, Seconds(65));
+                        events.ScheduleEvent(RP1_EVENT_JAINA1, Seconds(78));
+                        events.ScheduleEvent(RP1_EVENT_ARTHAS8, Seconds(80));
+                        events.ScheduleEvent(RP1_EVENT_ARTHAS8_2, Seconds(86));
+                        events.ScheduleEvent(RP1_EVENT_UTHER6, Seconds(92));
+                        events.ScheduleEvent(RP1_EVENT_ARTHAS9, Seconds(97));
+                        events.ScheduleEvent(RP1_EVENT_JAINA2, Seconds(99));
+                        events.ScheduleEvent(RP1_EVENT_ARTHAS10, Seconds(124));
+                    }
+                    break;
+                case RP2_ARTHAS_MOVE_1:
+                    if (Creature* citizen = me->FindNearestCreature(NPC_CITIZEN, 100.0f, true))
+                        citizen->SetFacingToObject(me);
+                    if (Creature* resident = me->FindNearestCreature(NPC_RESIDENT, 100.0f, true))
+                        resident->SetFacingToObject(me);
+                    events.ScheduleEvent(RP2_EVENT_CITIZEN1, Seconds(2));
+                    events.ScheduleEvent(RP2_EVENT_ARTHAS2, Seconds(10));
+                    events.ScheduleEvent(RP2_EVENT_ARTHAS_MOVE_2, Seconds(11));
+                    break;
+                case RP2_ARTHAS_MOVE_2:
+                    if (Creature* citizen = me->FindNearestCreature(NPC_CITIZEN, 100.0f, true))
+                    {
+                        DoCast(citizen, SPELL_CRUSADER_STRIKE);
+                        me->Kill(citizen);
```

Comment:
actually the question is more directed into which is the packet sent, damage, update..?